.github/workflows/nightly-pipeline.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
           github_token: ${{ secrets.WORKFLOW_TOKEN }}
           github_user: GPUtester
           workflow_file_name: build.yaml
-          ref: ${{ fromJSON(needs.get-run-info.outputs.obj).branch }}
+          ref: main
```

Comment:
I think this should still be computed, instead of hardcoded as `main`.

During code freeze, I think we are going to run nightly CI on `release/YY.MM` instead of `main` and we need a way to change that over. I was guessing that would happen by changing the output of the `get-run-info`?

(You have probably thought about this more than me, so ignore if this is solved some other way.)